dev/integration_tests/android_semantics_testing/lib/src/constants.dart

```diff
@@ -30,147 +28,97 @@ class AndroidClassName {
 }
 
 /// Action constants which correspond to `AccessibilityAction` in Android.
-@immutable
-class AndroidSemanticsAction {
-  const AndroidSemanticsAction._(this.id);
-
-  /// The Android id of the action.
-  final int id;
-
-  static const int _kFocusIndex = 1 << 0;
-  static const int _kClearFocusIndex = 1 << 1;
-  static const int _kSelectIndex = 1 << 2;
-  static const int _kClearSelectionIndex = 1 << 3;
-  static const int _kClickIndex = 1 << 4;
-  static const int _kLongClickIndex = 1 << 5;
-  static const int _kAccessibilityFocusIndex = 1 << 6;
-  static const int _kClearAccessibilityFocusIndex = 1 << 7;
-  static const int _kNextAtMovementGranularityIndex = 1 << 8;
-  static const int _kPreviousAtMovementGranularityIndex = 1 << 9;
-  static const int _kNextHtmlElementIndex = 1 << 10;
-  static const int _kPreviousHtmlElementIndex = 1 << 11;
-  static const int _kScrollForwardIndex = 1 << 12;
-  static const int _kScrollBackwardIndex = 1 << 13;
-  static const int _kCutIndex = 1 << 14;
-  static const int _kCopyIndex = 1 << 15;
-  static const int _kPasteIndex = 1 << 16;
-  static const int _kSetSelectionIndex = 1 << 17;
-  static const int _kExpandIndex = 1 << 18;
-  static const int _kCollapseIndex = 1 << 19;
-  static const int _kSetText = 1 << 21;
+enum AndroidSemanticsAction {
```

Comment:
I wonder why the `use_enum` lint didn't catch this one...

https://dart.dev/tools/linter-rules#use_enums

Reply:
cc @pq

Reply:
Hmmm. Looking. Thanks!